### ISSUE TITLE
Improve initial sync phase UX

### ISSUE THREAD

**Issue by fab-10** [hyperledger/besu#3863]
### Description
As an first time Besu user, I want to have informative logs so that I can understand how long it takes to complete the initial sync. 

### Acceptance Criteria
* Besu should clearly show the progess of the initial sync in the logs

**Expected behavior:** [What you expect to happen]

We I start Besu for the first time, I want to easily understand how the initial sync is going by looking at the logs.

The optimal information is something like that: 
**Initial sync: 30% done, ETA: 18h 29m**
but if that is not possible, because we still do not know the total amount of work to be done, it is acceptable to show a progress in term of the work done till now:
**Downloaded x world state nodes, and y blocks"

These logs are meant to be consumed by human, to they should be print at regular interval, not too fast, but also not to slowly to avoid givin the false idea that Besu is doing nothing, for example every minute could be a good tradeoff.

During the initial sync phase there should be no other logs at info level that pollute the logs and create confusion to the user, of course warnings and errors are always allowed.

**Actual behavior:** [What actually happens]

Currently many first time user are confused by Besu logs during the initial sync, since there are some logs that are not informative, like:
```
DNSTimerTask [Timer-0] Refreshing DNS records with enrtree://AKA3AM6LPBYEUDMVNU3BSVQJ5AD45Y7YPOHJLEF6W26QOE4VTUDPE@all.mainnet.ethdisco.net
```
or
```
SyncTargetManager | Found common ancestor with peer Peer 0x8ba0932c425635a9f7... PeerReputation 100, validated? true, disconnected? false at block 14785691
```
that are logged frequently and usually hide the other more informative logs about the number of blocks and world state nodes downloaded.

**Comment by diglos** [comment]
I agree, particularly with the second one. It is very confusing. At first it seems that the sync is stucked and something is wrong.

**Comment by fab-10** [comment]
There are a couple of option to manage the DNSTimerTask log, the simple one is to just tune our default log4j configuration to silence it, the other option is to propose a change to the Tuweni library and make its log configurable.

For the world state and the block downloads, instead we need to rethink how we show progress, and hopefully unify it, so we can have a better ETA of the whole initial sync progress, since the average user should not be interested about the different progress on block download or state download on themselves.

@gezero since you are doing a refactoring of the initial sync, it could be a good opportunity to rethink also how we show progress to the user.

**Comment by matt-nelson-csi** [comment]
User's have also mentioned that the initial sync should echo some/all of their configuration options (without exposing Auth) to confirm what sync mode, storage mode, and more they have chosen. I think this could be useful as Besu initializes to re-assure users their configuration was done properly (or that it was not and they should reinitialize sync). I also think we should print logs with regards to the consensus layer to target a post-Merge context as well. I.e. a sync will fail to complete on a post-merge network with a graceful error catch if there is no CL endpoint configured. The CL prints that it is waiting for the EL to sync, so clear messages about this handoff would be appreciated for new users as well.

**Comment by fab-10** [comment]
@matt-nelson-csi [this ticket](https://github.com/hyperledger/besu/issues/3864) is related to your comment.

**Comment by PolestarWx** [comment]
So no improvement planed, no ETA indicator ?
